front/app/components/StatusModule/index.tsx

```diff
@@ -125,8 +125,6 @@ const StatusModule = ({ votingMethod, phase, project }: StatusModuleProps) => {
           </Box>
           {phase && showDate && phase.attributes.end_at && (
             <Text>
-              {config?.getSubmissionTerm &&
-                formatMessage(config.getSubmissionTerm('plural'))}{' '}
               {formatMessage(messages.submittedUntil)}{' '}
```

Comment:
I actually think it makes more sense to remove the plural submission term here and just use "Your vote" anyways, as it's what we're using elsewhere in the copy (E.g. We're not actually submitting _tokens_ - we're submitting a _vote_ where we've assigned tokens).

Instead of "Tokens may be submitted until {date}", the copy is now "Your vote may be submitted until {date}".

Reply:
Sounds good!